scripts/install/authelia.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#
```

Comment:
This whole file should be re-written with segments of code separated into functions.

That way we can also make a simple implementation for MU-later, which will only call some portions of this when the install script is being invoked with an argument which will hold the name of the new user being added